### ISSUE TITLE
AlmaDownloadHandler import error

### ISSUE THREAD

**Issue by astrojysun** [akleroy/phangs_imaging_scripts#226]
I got the following issue after updating my repo to the latest merge:

```
/home/sun208/workspace/python/phangs_imaging_scripts/phangsPipeline/__init__.py in <module>()
     13 from .handlerKeys import KeyHandler
     14 from .handlerSingleDish import SingleDishHandler
---> 15 from .handlerAlmaDownload import AlmaDownloadHandler
     16 from .handlerVis import VisHandler
     17 from .handlerPostprocess import PostProcessHandler

ImportError: cannot import name AlmaDownloadHandler
```

Seems like it is due to a mis-indentation issue here: https://github.com/akleroy/phangs_imaging_scripts/blob/master/phangsPipeline/handlerAlmaDownload.py#L245

**Comment by thomaswilliamsastro** [comment]
Hmm, I think the indentation is intentional (so it doesn't break without packages installed and to maintain compatibility), but I think wrapping

```
try:
    from .handlerAlmaDownload import AlmaDownloadHandler
except ImportError:
    pass
```

may be the solution here

**Comment by thomaswilliamsastro** [comment]
Sounds good to me!

**Comment by thomaswilliamsastro** [comment]
This is fixed in #238
